infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutDesignDao.kt

```diff
@@ -41,13 +42,13 @@ class LayoutDesignDao(
         }
     }
 
-    fun list(): List<LayoutDesign> {
+    fun list(includeCompletedAndDeleted: Boolean = false): List<LayoutDesign> {
         val sql = """
             select id, name, estimated_completion, design_state
             from layout.design
-            where design_state = 'ACTIVE'::layout.design_state
+            where design_state = 'ACTIVE'::layout.design_state or :include_completed_and_deleted is true
```

Comment:
Toistaiseksi listauksessa on tasan kaksi tapausta:
1. Vain aktiiviset
2. Kaikki

Tämän takia en katsonut järkeväksi erottaa näitä ehtoja toisistaan, vaan tehdä vain yhden parametrin. Erotetaan sitten jos/kun joskus halutaan listata vaikka aktiiviset ja valmiit.